pkg/models/monitoring/monitoring.go

```diff
@@ -89,21 +91,49 @@ func (mo monitoringOperator) GetMetricOverTime(expr, namespace string, start, en
 			return monitoring.Metric{}, err
 		}
 	}
-	return mo.c.GetMetricOverTime(expr, start, end, step), nil
+	return mo.prometheus.GetMetricOverTime(expr, start, end, step), nil
 }
 
 func (mo monitoringOperator) GetNamedMetrics(metrics []string, time time.Time, opt monitoring.QueryOption) Metrics {
-	ress := mo.c.GetNamedMetrics(metrics, time, opt)
+	ress := mo.prometheus.GetNamedMetrics(metrics, time, opt)
+	res_metrics := mo.metricsserver.GetNamedMetrics(metrics, time, opt)
+
+	//Merge edge node metrics data
+	metricsEdge := make(map[string]monitoring.MetricData)
```

Comment:
looks like edgeMetrics is better than metricsEdge

Reply:
Fixed